src/kem/kyber/ind_cpa.rs

```diff
@@ -211,6 +211,10 @@ pub(crate) fn encrypt<
     const VECTOR_U_COMPRESSION_FACTOR: usize,
     const VECTOR_V_COMPRESSION_FACTOR: usize,
     const BLOCK_LEN: usize,
+    const ETA1: usize,
+    const ETA1_RANDOMNESS_SIZE: usize,
+    const ETA2: usize,
+    const ETA2_RANDOMNESS_SIZE: usize,
```

Comment:
eta2 is always the same so this isn't necessary. But let's keep in case it changes. Two more doesn't hurt at this point 😆

Reply:
Heh yeah, I also thought I'd include it in to help document the code according to the specification.